Source/JavaScriptCore/API/glib/JSCOptions.cpp

```diff
@@ -651,18 +651,18 @@ void jsc_options_foreach(JSCOptionsFunc function, gpointer userData)
 #undef VISIT_OPTION
 }
 
-WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN // GLib port
 static gboolean setOptionEntry(const char* optionNameFull, const char* value, gpointer, GError** error)
 {
-    const char* optionName = optionNameFull + 6; // Remove the --jsc- prefix.
+    std::span<const char> fullNameSpan(optionNameFull, strlen(optionNameFull));
+    constexpr size_t prefixLength = 6;
+    const char* optionName = fullNameSpan.subspan(prefixLength).data();
```

Comment:
The `// Remove the --jsc- prefix.` comment was pretty useful IMO but got dropped.